src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -312,7 +312,8 @@ export function MakeEnrichmentsTabUI(
     getStore:()=>GroupComparisonStore,
     getEnrichmentsUI:()=>MobxViewAlwaysComponent,
     enrichmentType:string,
-    multiGroupAnalysisPossible?:boolean
+    multiGroupAnalysisPossible?:boolean,
+    multiStudyAnalysisPossible?:boolean
```

Comment:
will this be necessary when you support multi-study for mrna and protein?

Reply:
not sure. anyways multiStudyAnalysisPossible is set to false for them